include/deal.II/matrix_free/cuda_matrix_free.templates.h

```diff
@@ -732,6 +755,18 @@ namespace CUDAWrappers
 
 
 
+  template <int dim, typename Number>
+  template <typename functor>
+  void
+  MatrixFree<dim, Number>::evaluate_coefficients(functor &func) const
```

Comment:
Hmm... here we use a reference while all the other places use copies.

Reply:
This is fine because the function is only on the cpu.